reconcile/kafka_clusters.py

```diff
@@ -102,14 +119,15 @@ def run(dry_run, thread_pool_size=10,
             error = True
             continue
         # check if cluster is ready. if not - wait
-        if current_cluster['status'] != 'complete':
+        if current_cluster['status'] != 'ready':
```

Comment:
Can we convert these magic strings into constants? A mistake on a future iteration, (say, I'm pushing this during an outage and I type `rready`) will waste hours of debugging.

Reply:
done